charts/plex-media-server/values.yaml

```diff
@@ -0,0 +1,224 @@
+# Default values for pms-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# The docker image information for the pms application
+
+image:
+  registry: index.docker.io
+  repository: plexinc/pms-docker
+  # If unset use "latest"
+  tag: 1.31.0.6654-02189b09f
```

Comment:
As discussed, we should use `latest` or leave it empty so we don't end up shipping an older (maybe vulnerable) version of PMS.